week-8/Homework/mandatory/3-dog-photo-gallery/main.js

```diff
@@ -0,0 +1,28 @@
+let root = document.getElementById("root");
+root.style.width = "100%";
+let button = document.createElement("button");
+root.appendChild(button).innerText = "New dog";
+button.addEventListener("click", randomDog);
+let ul = document.createElement("ul");
+root.appendChild(ul);
+ul.setAttribute("style", "display: flex; flex-wrap: wrap");
+
+function randomDog() {
+  fetch("https://dog.ceo/api/breeds/image/random")
+    .then((response) => {
+      if (!response.ok) {
+        throw Error(response.statusText);
+      }
+      return response.json();
+    })
+    .then((data) => {
+      let li = document.createElement("li");
+      li.style.listStyleType = "none";
```

Comment:
We can set the list style type as none for the <ul> element rather than for every <li> we add to the list. This can be done in a CSS file rather than in JS, as I mentioned above :D

Reply:
Yes, and it's more clearly to see.